.github/workflows/build.yaml

```diff
@@ -20,7 +23,7 @@ jobs:
         run: |
           git submodule init
           git submodule update --recursive --progress --recommend-shallow
-
+   
```

Comment:
Nit: remove the blanks.

Reply:
https://github.com/riscv/riscv-gnu-toolchain/commit/407cdc0ceb0f6d760342ac6cf1b890378f5a81bd